lib/impl/testing_utils.dart

```diff
@@ -0,0 +1,36 @@
+
+/// Return the list of possible test paths. The given [path] must be in the
+/// `lib/` directory.
+List<String> getPossibleTestPaths(String path, { String separator: '/' }) {
```

Comment:
Perhaps I can make this an assert != null? I'm working around an issue here where this library can't import any of the node wrappers directly, as those import dart:html, and I want this code to be able to be run by CLI unit tests. It's a fair bit of indirection :)